14-EventToCommandBehavior/src/PrismSample/ViewModels/MainPageViewModel.cs

```diff
@@ -1,11 +1,41 @@
-﻿using System;
-using System.Collections.Generic;
-using System.Text;
+﻿using PrismSample.Views;
+using Prism.Navigation;
+using Prism.Commands;
+using Xamarin.Forms;
 using Prism.Mvvm;
 
 namespace PrismSample.ViewModels
 {
     public class MainPageViewModel : BindableBase
     {
+        private readonly INavigationService _navigationService;
+
+        public DelegateCommand GoToEventArgsConverterExamplePageCommand { get; private set; }
+        public DelegateCommand GoToEventArgsParameterExamplePageCommand { get; private set; }
+        public DelegateCommand GoToSimpleExamplePageCommand { get; private set; }
+
+        public MainPageViewModel(INavigationService navigationService)
+        {
+            _navigationService = navigationService;
+
+            GoToEventArgsConverterExamplePageCommand = new DelegateCommand(GoToEventArgsConverterExamplePage);
+            GoToEventArgsParameterExamplePageCommand = new DelegateCommand( GoToEventArgsParameterExamplePage);
+            GoToSimpleExamplePageCommand = new DelegateCommand(GoToSimpleExamplePage);
+        }
+
+        private async void GoToEventArgsConverterExamplePage()
+        {
+            await _navigationService.NavigateAsync(nameof(NavigationPage) + "/" + nameof(EventArgsConverterExamplePage));
+        }
+
+        private async void GoToEventArgsParameterExamplePage()
+        {
+            await _navigationService.NavigateAsync(nameof(NavigationPage) + "/" + nameof(EventArgsParameterExamplePage));
+        }
+
+        private async void GoToSimpleExamplePage()
+        {
+            await _navigationService.NavigateAsync(nameof(NavigationPage) + "/" + nameof(SimpleExamplePage));
+        }
```

Comment:
This 100% breaks MVVM when you use `nameof(SomePage)` in a ViewModel. Also the navigation pattern here makes for a poor experience where you have to stopped the app and rerun to see another example... update these just to the name of the page you want and update the navigation in App.OnInitialized to use `NavigationPage/MainPage`

Reply:
Done with the latest commit. :)